.github/workflows/dependencies.yml

```diff
@@ -10,11 +10,10 @@ on:
     - cron: '18 23 * * 3'
 
 env:
-  IS_PUSHING_IMAGES: ${{ github.event_name != 'pull_request' && github.repository == 'photoview/photoview' }}
+  #IS_PUSHING_IMAGES: ${{ github.event_name != 'pull_request' && github.repository == 'photoview/photoview' }}
+  IS_PUSHING_IMAGES: true
   IS_CACHING: ${{ github.event_name == 'pull_request' }}
-  DOCKER_USERNAME: viktorstrate
-  DOCKER_IMAGE: viktorstrate/dependencies
-  DOCKER_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
+  DOCKER_IMAGE: ghcr.io/photoview/dependencies
```

Comment:
I guess it should be `ghcr.io/photoview/photoview/dependencies`, as it's a image of `photoview/photoview` repo.

Reply:
I've tried this change: https://github.com/photoview/photoview/actions/runs/11559527551/job/32174376849?pr=1108#step:7:23138
But it also doesn't work, as this is not a regular repo, but part of the `photoview` org. GitHub registry for images and packages is on the org level, not on the repo level for org repos, so the initial address was correct.
Anyway, I wanted to double-check it

Reply:
Did you grant the writing permission to workflows in Settings? 
![image](https://github.com/user-attachments/assets/7a8c8d0e-10a5-4d3d-a5d8-f7ad8f489053)

Reply:
I don't have admin permissions in this org(
@viktorstrate, this question to you

Reply:
I'd like to try the simple one first: `ghcr.io/photoview/photoview:dependencies` or even `ghcr.io/photoview/photoview`. If all settings are right, these should work well. After that, we can check how a sub-image works.

Reply:
```
Error: buildx failed with: ERROR: failed to solve: failed to push ghcr.io/photoview/photoview/photoview:latest: unexpected status from POST request to https://ghcr.io/v2/photoview/photoview/photoview/blobs/uploads/: 403 Forbidden
```
So, basically, we cannot use the GHCR of the Photoview org.

Reply:
I see you change the image name to `ghcr.io/photoview/photoview/photoview`. Is that right? I think it should be 'ghcr.io/photoview/photoview`.

Reply:
I tried that earlier with the `dependencies` image. If GHCR accepts login and allows pushing, it should be a different error message

Reply:
I've successfully pushed the image to the Docker hub org registry with my temporary PAT used. I've asked @viktorstrate in the Discord to double-check what is inside the current secret (PAT or password), as only the PAT is fully-supported.